platforms/mixin-common/src/main/java/com/dfsek/terra/mod/mixin/implementations/terra/block/entity/MobSpawnerBlockEntityMixin.java

```diff
@@ -50,15 +50,15 @@ private MobSpawnerBlockEntityMixin(BlockEntityType<?> type, BlockPos pos, BlockS
     public abstract MobSpawnerLogic getLogic();
 
     @Shadow
-    public abstract void setEntityType(net.minecraft.entity.EntityType<?> entityType, Random random);
+    public abstract void method_46408(net.minecraft.entity.EntityType<?> entityType, Random random);
```

Comment:
yarn mappings removed this name?

should we open an issue for this?

Reply:
yes. that might be a good idea